CMakeLists.txt

```diff
@@ -67,9 +68,14 @@ macro(iotjs_add_link_flags)
   iotjs_add_flags(IOTJS_LINKER_FLAGS ${ARGV})
 endmacro()
 
+CHECK_C_COMPILER_FLAG("-no-pie" HAS_NO_PIE)
```

Comment:
I suggest removing the double-quotes here like `CHECK_C_COMPILER_FLAG(-no-pie HAS_NO_PIE)`

Reply:
Thank you, I updated the PR as you suggested.